app/assets/stylesheets/buttons.scss

```diff
@@ -1,25 +1,52 @@
 .btn {
   display: inline-block;
-  padding: 0.5rem 1rem;
+  font-size: 0.85rem;
   border-radius: 0.25rem;
   font-weight: 500;
   text-align: center;
   border: none;
+  color: inherit;
+  background-color: transparent;
   transition: translate 150ms ease-in-out, background-color 150ms ease-in-out;
 
-  &:hover {
-    translate: 0 -0.125rem;
-    background-color: var(--dark-grey);
-    box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.25);
-  }
+  &_cta {
+    --hover: var(--dark-grey);
+    --active: var(--dark);
+    --color: var(--white);
+    --bg-color: var(--dark);
+
+    padding: 0.5rem 1rem;
+    color: var(--color);
+    background-color: var(--bg-color);
+
+    &:hover {
+      translate: 0 -0.125rem;
+      background-color: var(--hover);
+      box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.25);
+    }
 
-  &:active {
-    translate: 0 0;
-    background-color: var(--dark);
+    &:active {
+      translate: 0 0;
+      background-color: var(--active);
+    }
+
+    &_alt {
+      --color: var(--dark);
+      --bg-color: var(--white);
+    }
+
+    &_new {
+      --hover: var(--green);
+      --active: var(--green);
+    }
```

Comment:
- [ ] Great UI and design concept for the application. I can see that you took your time to learn the framework 👏 💪

Reply:
Thank you@Gambit142😃.